internal/pkg/apps/install.go

```diff
@@ -124,6 +152,9 @@ func Install(ctx context.Context, clients *shared.ClientFactory, log *logger.Log
 		app.AppID = result.AppID
 		app.TeamID = *authSession.TeamID
 		app.TeamDomain = auth.TeamDomain
+		// TODO: add enterprise ID and user ID to app? See InstallLocalApp.
+		// app.EnterpriseID = config.GetContextEnterpriseID(ctx)
+		// app.UserID = *authSession.UserID
```

Comment:
note: While comparing the dev install and this install, I noticed that we set the `EnterpriseID` and `UserID` in the other one. I've left a note incase we want to do it here as well.

Reply:
@mwbrooks `EnterpriseID` makes a lot of sense to me!

I'm hesitant for the `UserID` at the moment since we might be using this - alongside the `IsDev` value - to differentiate "deployed" and "local" apps IIRC 🔍 

No blocker for this PR, but the app `UserID` might be something to inspect more later!

Reply:
@zimeg I've removed the `UserID` comment so that we don't accidentally implement it later on. I agree, there could be bits of code relying on it to identify a dev app.